tests/integration/dotnet/basic/Program.cs

```diff
@@ -0,0 +1,317 @@
+﻿// Copyright 2016-2019, Pulumi Corporation.  All rights reserved.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Pulumi;
+using Pulumi.Kubernetes.Core.V1;
+using Pulumi.Kubernetes.Types.Inputs.Core.V1;
```

Comment:
Is there any way to avoid having to import all of these individually?

Reply:
No - at least not without typing them out explicitly when used.